quixstreams/sources/community/file/origins/local.py

```diff
@@ -0,0 +1,36 @@
+from io import BytesIO
+from pathlib import Path
+from typing import Generator, Optional
+
+from .base import FileOrigin
+
+__all__ = ("LocalFileOrigin",)
+
+
+class LocalFileOrigin(FileOrigin):
+    def __init__(
+        self,
+    ):
+        self._client = None
+        self._credentials = {}
+        self.root_location = "/"
+
+    @property
+    def client(self):
+        return
```

Comment:
This and `self._client = None` and both unused. Seems redundant 😅

What I'm trying to say is that maybe base class doesn't need to be aware that some implementations _may_ need a client?

Reply:
(this is being discussed in another thread)